css/form.css

```diff
@@ -367,7 +373,7 @@ html[dir="rtl"] .checkbox.checkbox-icon input[type="checkbox"] + label > span.ch
   transition: all 0.25s ease;
 }
 
-.checkbox.checkbox-icon input[type="checkbox"]:checked + label span.check > i.fa {
+.checkbox.checkbox-icon input[type="checkbox"]:checked+label span.check>i.fa {
```

Comment:
Pretty sure spaces were meant to stay here. Without it makes it very difficult to read the rule.

Reply:
@squallstar Corrected. The spaces were being removed by the beautifier's default settings.